src/probnum/filtsmooth/gaussfiltsmooth/kalmanposterior.py

```diff
@@ -0,0 +1,96 @@
+"""
+Posterior over states after applying (Extended/Unscented) Kalman filtering/smoothing
+
+Contains the discrete time and function outputs.
+Provides dense output by being callable.
+Can function values can also be accessed by indexing.
+"""
+from probnum.prob.randomvariablelist import _RandomVariableList
+from probnum.filtsmooth.filtsmoothposterior import FiltSmoothPosterior
+
+
+class KalmanPosterior(FiltSmoothPosterior):
+    """Posterior Distribution after (Extended/Unscented) Kalman Filtering/Smoothing"""
+
```

Comment:
Add example / doctest

Reply:
Don't really know where to find, or how to write, a good example for this. 
@pnkraemer Do you have a fitting example I could use?
Otherwise I will probably not resolve this for this PR.

Reply:
see my other comment in the KalmanPosterior tests: merge anyway but keep this around as an 'improvement' issue? I would then take a look in a separate PR.

Reply:
Sounds good to me!